docs/getting-started-guides/clc.md

```diff
@@ -7,12 +7,23 @@ These scripts handle the creation, deletion and expansion of kubernetes clusters
 
 You can accomplish all these tasks with a single command. We have made the Ansible playbooks used to perform these tasks available [here](https://github.com/CenturyLinkCloud/adm-kubernetes-on-clc/blob/master/ansible/README.md).
 
+## Latest Changes
+
+Our latest release (v0.5) included the following major changes:
+
+- Kubernetes 1.2 versus 1.1
+- Integrated Load Balancing: Ie, kubernetes services of type 'LoadBalancer' now automatically get a public IP address routed to all minions in the cluster via the Centurylink Cloud LB service.
```

Comment:
- Integrated Load Balancing, i.e. Kubernetes services of type `LoadBalancer` now automatically get a public IP address routed to all minions in the cluster via the CenturyLink Cloud LB service.